[model_cards] Migrate cards from this repo to model repos on huggingface.co (#9013)
huggingface/transformers: docs/source/model_sharing.rst

In order to upload a model, you'll need to first create a git repo. This repo will live on the model hub, allowing
users to clone it and you (and your organization members) to push to it.

> You can create a model repo **directly from `the website <https://huggingface.co/new>`__.**

Author: ```suggestion
You can create a model repo **directly from `the /new page on the website <https://huggingface.co/new>`__.**
```

> You can create a model repo **directly from the website, `here <https://huggingface.co/new>`__.**

Reviewer: ```suggestion
You can create a model repo **directly from `the website <https://huggingface.co/new>`__.**
```

    echo "hello" >> README.md
    git add . && git commit -m "Update from $USER"

> We are intentionally not wrapping git too much, so that you can pick the workflow that you already know.

Reviewer: ```suggestion
We are intentionally not wrapping git too much, so that you can go on with the workflow you're used to and the tools you already know.
```

The only learning curve you might have compared to regular git is the one for git-lfs. The documentation at

> https://git-lfs.github.com/ is decent, but we'll work on a tutorial with some tips and tricks in the coming weeks!

Author: cc @Pierrci (with stuff like how to get a progress bar when cloning, etc.)

> https://git-lfs.github.com/ is decent, but we'll work on a tutorial with some tips and tricks in the coming weeks!

Reviewer: Make a link.

> https://git-lfs.github.com/ is decent, but we'll work on a tutorial with some tips and tricks in the coming weeks!

Author: ```suggestion
`git-lfs.github.com <https://git-lfs.github.com/>`__ is decent, but we'll work on a tutorial with some tips and tricks in the coming weeks!
```

Add a model card
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> To make sure everyone knows what your model can do, what its limitations and potential bias or ethical considerations,

Reviewer 2: ```suggestion
To make sure everyone knows what your model can do, what its limitations, potential bias or ethical considerations are,
```

please add a README.md model card to your model repo. You can just create it, or there's also a convenient button

> titled "Add a README.md" on your model page. If you need one, `here <https://github.com/huggingface/model_card>`__ is a

Reviewer 2: It's currently called "Add readme" I think (the button)

> titled "Add a README.md" on your model page. If you need one, `here <https://github.com/huggingface/model_card>`__ is a

Reviewer 2: ```suggestion
titled "Add a README.md" on your model page. A model card template can be found  `here <https://github.com/huggingface/model_card>`__ (meta-suggestions are welcome).
```

    sudo apt-get install git-lfs

> Then you can use either create a repo directly from the huggingface.co website, or use the :obj:`transformers-cli` to

Reviewer 2: ```suggestion
Then you can use either create a repo directly from  `huggingface.co <https://huggingface.co/>`__ , or use the :obj:`transformers-cli` to
```
